proteus/mesh.pxd

```diff
@@ -155,6 +155,7 @@ cdef extern from "mesh.h":
                                                                     const double& Ly,
                                                                     const double& Lz,
                                                                     Mesh& mesh)
+    cdef int reorientTetrahedralMesh(Mesh& mesh)
```

Comment:
could you briefly describe what reorient means here?

Reply:
it checks the sign of the Jacobian determinant for each tet and swaps the order of two nodes in the elementNodesArray if the determinant is negative. Most mesh generators already do that, but I found that the simple 3D tet from regular grid algorithm was not. Proteus generally doesn't care, but Paraview uses it to decide between "front faces" and "back faces".